data_owner_ids.py

```diff
@@ -18,15 +23,51 @@ def parse_args():
     return args
 
 
-def process_csv(csv_path, system_csv_path, system):
+def extract_metadata_and_zip(system, system_zip_path, system_output_dir_path, n_rows):
+    print(str(system_output_dir_path))
```

Comment:
Do we want to leave this print statement in? Seems like it's for debugging.

Reply:
I can't come up with a reason for it being there other than debugging, so out it goes!